src/ui/common/src/components/layouts/NavBar.tsx

```diff
@@ -0,0 +1,219 @@
+import { faBell, faCircleUser } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  AppBar,
+  Avatar,
+  Breadcrumbs,
+  Link,
+  Menu,
+  MenuItem,
+  Toolbar,
+  Typography,
+} from '@mui/material';
+import Box from '@mui/material/Box';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Link as RouterLink } from 'react-router-dom';
+
+import { RootState } from '../../stores/store';
+import { theme } from '../../styles/theme/theme';
+import UserProfile from '../../utils/auth';
+import { getPathPrefix } from '../../utils/getPathPrefix';
+import {
+  NotificationLogLevel,
+  NotificationStatus,
+} from '../../utils/notifications';
+import NotificationsPopover from '../notifications/NotificationsPopover';
+import styles from './menu-sidebar-styles.module.css';
+import { MenuSidebarWidthNumber } from './menuSidebar';
+
+const pathPrefix = getPathPrefix();
+
+export class BreadcrumbLinks {
```

Comment:
Is there a particular reason why this is a class?

We're using functional components throughout the entire project.

Reply:
I was trying to do an enum with display name & link for the breadcrumb links to be used in the breadcrumbs but enums are only for strings & numbers, I believe and followed this stackoverflow post: https://stackoverflow.com/questions/41179474/use-object-literal-as-typescript-enum-values/51398471#51398471

Let me know what you think would be the best way to implement this